torch/csrc/jit/python/init.cpp

```diff
@@ -1627,6 +1630,8 @@ void initJITBindings(PyObject* module) {
       [](const char* op_name, py::args args, py::kwargs kwargs) {
         try {
           auto symbol = Symbol::fromQualString(op_name);
+          bool allow_numbers_as_tensors = opAllowsNumbersAsTensors(symbol);
+          ToIValueAllowNumbersAsTensors g(allow_numbers_as_tensors);
```

Comment:
This is a bugfix: previously, `torch._C._jit_resolve_packet` was inconsistent with how OpOverloadPacket resolve overloads (the inconsistency is that some ops are allowed to promote numbers to Tensors). These two lines fix that inconsistency.